lib/commands/simulate.ts

```diff
@@ -13,23 +13,13 @@ export class SimulateCommand implements ICommand {
 	}
 
 	public execute(args: string[]): IFuture<void> {
-		return this.$simulatorService.launchSimulator();
+		return (() => {
+			if (this.$simulatorPlatformServices && this.$simulatorPlatformServices.canRunApplication().wait()) {
```

Comment:
`this.$simulatorPlatformServices` is empty object, so  `if(this.$simulatorPlatformServices)` is true. You should use:
```TypeScript
if (this.$options.simulator && this.$simulatorPlatformServices.canRunApplication && this.$simulatorPlatformServices.canRunApplication().wait()) {
```